src/ORM/Behavior/TimestampBehavior.php

```diff
@@ -136,8 +136,8 @@ public function implementedEvents()
      */
     public function timestamp(DateTime $ts = null, $refreshTimestamp = false)
     {
-        if ($ts) {
-            if ($this->_config['refreshTimestamp']) {
+        if ($ts instanceof DateTime) {
```

Comment:
Why is this change needed? The $ts paramater is typed already type hinted as DateTime so type check again is not necessary.